packages/shared/src/modules/users/libs/types/user-sign-up-submit-dto.type.ts

```diff
@@ -0,0 +1,8 @@
+type UserSignUpSubmitDto = {
```

Comment:
Check #168 - this type was already implemented there, we should have the same so that we don't have 2 same types later